packages/decision-tree/src/components/Question/Question.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { type QuestionProps } from './types';
+import { Flex } from '@cmpsr/components';
+import { Height, SingleChoice, MultipleChoice, Numeric } from './questionTypes';
+
+const questionTypesMap = {
+  height: ({ data, answersDispatch }) => <Height data={data} answersDispatch={answersDispatch} />,
+  singleChoice: ({ data, answersDispatch }) => <SingleChoice data={data} answersDispatch={answersDispatch} />,
+  multipleChoice: ({ data, answersDispatch }) => <MultipleChoice data={data} answersDispatch={answersDispatch} />,
+  numeric: ({ data, answersDispatch }) => <Numeric data={data} answersDispatch={answersDispatch} />,
+};
+
+export const Question: FC<QuestionProps> = ({ data, answersDispatch }) => {
+  if (!questionTypesMap[data.type]) return null;
+  if (typeof questionTypesMap[data.type] !== 'function') return null;
```

Comment:
can this be simplified to? I think that the second if catch both
```suggestion
  if (typeof questionTypesMap[data.type] !== 'function') return null;
```

Reply:
Actually I had this for a reason that does not make sense anymore, I'll remove the checks.